lib/Sema/TypeCheckStmt.cpp

```diff
@@ -947,6 +947,142 @@ class StmtChecker : public StmtVisitor<StmtChecker, Stmt*> {
     return S;
   }
 
+  void checkCaseLabelItem(CaseStmt *caseBlock, CaseLabelItem &labelItem,
+                          bool &limitExhaustivityChecks, Type subjectType) {
+    SWIFT_DEFER {
```

Comment:
I am actually going to rip out the usage of SWIFT_DEFER here in a subsequent commit and hoist it back into visitSwitchStmt. I just don't want to stop the testing now.